app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,19 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self,
+                 nickname: str,
+                 favourite_dish: str,
+                 hummer_level: int
```

Comment:
There seems to be a typo in the parameter name `hummer_level`. It should likely be `hammer_level` to accurately describe the attribute related to a warrior's hammer.